logstash-core/spec/logstash/agent_spec.rb

```diff
@@ -296,6 +296,24 @@
     end
   end
 
+  context "--config-test" do
```

Comment:
I would change this to `describe` as is more of a general are with two specific context for ```--config-test```